bin/migrate-oats-data/submissions/app_submissions.py

```diff
@@ -0,0 +1,262 @@
+    - Initializes empty lists
+    - Iterates over 'app_sub_raw_data_list'
+        - Maps the basic fields of the data dictionary based on the alr_change_code
+    - Returns the mapped lists
+    """
+    nfu_data_list = []
+    nar_data_list = []
+    exc_data_list = []
+    inc_data_list = []
+    other_data_list = []
+
+    for row in app_sub_raw_data_list:
+        data = dict(row)
+        # data = map_basic_field(data)
+
+        if data["alr_change_code"] == ALRChangeCode.NFU.value:
+            # data = mapOatsToAlcsAppPrep(data)
+            nfu_data_list.append(data)
+        elif data["alr_change_code"] == ALRChangeCode.NAR.value:
+            nar_data_list.append(data)
+        elif data["alr_change_code"] == ALRChangeCode.EXC.value:
+            # data = mapOatsToAlcsLegislationCode(data)
+            exc_data_list.append(data)
+        elif data["alr_change_code"] == ALRChangeCode.INC.value:
+            # data = mapOatsToAlcsLegislationCode(data)
+            inc_data_list.append(data)
+        else:
+            other_data_list.append(data)
+
+    return nfu_data_list, nar_data_list, other_data_list, exc_data_list, inc_data_list
+
+
+def get_update_query(unique_fields,unique_values):
+    # unique_fields takes input from called function and appends to query
+    query = """
+                INSERT INTO alcs.application_submission (
+                    file_number,
+                    local_government_uuid,
+                    type_code,
+                    is_draft,
+                    audit_created_by,
+                    applicant
+                    {unique_fields}
+                )
+                VALUES (
+                    %(file_number)s,
+                    %(local_government_uuid)s,
+                    %(type_code)s,
+                    false,
+                    'oats_etl',
+                    %(applicant)s
+                    {unique_values}
+                )
+    """
+    return query.format(unique_fields=unique_fields, unique_values=unique_values)
+
+def get_update_query_for_nfu():
+    unique_fields = """, 
+                    nfu_hectares
+    """
+    unique_values = """,
+                    %(alr_area)s
+    """
+    return get_update_query(unique_fields,unique_values)
+
+def get_update_query_for_nar():
+    # naruSubtype is a part of submission, import there
+    unique_fields = """"""
+    unique_values = """"""
+    return get_update_query(unique_fields,unique_values)
+
+
+def get_update_query_for_exc():
+    unique_fields = """,
+                    incl_excl_hectares
+    """
+    unique_values = """,
+                    %(alr_area)s
+    """
+    return get_update_query(unique_fields,unique_values)
+
+
+def get_update_query_for_inc():
+    unique_fields = """,
+                    incl_excl_hectares
+    """
+    unique_values = """,
+                    %(alr_area)s
+    """
+    return get_update_query(unique_fields,unique_values)
+
+
+def get_update_query_for_other():
```

Comment:
the same as above between NARU and Others. I think you do not need this functions at all since you could just call get_update_query() in prepare_app_sub_data()

Reply:
NARU has several unique fields to be mapped later